ci-operator/step-registry/ipi/conf/aws/usage-info/ipi-conf-aws-usage-info-commands.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+#set -o nounset
+#set -o errexit
+#set -o pipefail
+
+set +e
+
+env > "${ARTIFACT_DIR}/prow-ci-env.log"
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+cp ${CLUSTER_PROFILE_DIR}/pull-secret /tmp/pull-secret
+oc registry login --to /tmp/pull-secret
+ocp_version=$(oc adm release info --registry-config /tmp/pull-secret ${RELEASE_IMAGE_LATEST} -ojsonpath='{.metadata.version}' | cut -d. -f 1,2)
+ocp_major_version=$( echo "${ocp_version}" | awk --field-separator=. '{print $1}' )
+ocp_minor_version=$( echo "${ocp_version}" | awk --field-separator=. '{print $2}' )
+rm /tmp/pull-secret
+
+job_type="${JOB_TYPE:-}"
+user=""
+if [[ "${job_type}" == "presubmit" ]]; then
+    author=$(echo "${JOB_SPEC:-}" | jq -r '.refs.pulls[].author')
+    pull_number=${PULL_NUMBER:-unknown}
+    user="${author}-${pull_number}"
+fi
+if [[ "${job_type}" == "periodic" ]]; then
```

Comment:
Should we consider postsubmit too?

Reply:
when postsubmit job would be triggered ?

Reply:
Yes but also an else to catch any possible job type different than the ones considered?

Reply:
Do you have an example postsubmit job link, so that I can figure out what tag can be filled there.

Reply:
https://prow.ci.openshift.org/view/gs/test-platform-results/logs/branch-ci-openshift-assisted-service-release-ocm-2.10-mce-publish/1848258653547663360

Reply:
(I guess) they should be all the ones with `branch-ci` at the beginning.

Reply:
Done.